.github/workflows/react-native-sdk.yml

```
@@ -0,0 +1,115 @@
+                paths:
+                        - 'sdk/runanywhere-react-native/**'
+                        - '.github/workflows/react-native-sdk.yml'
+
+jobs:
+        lint-and-build:
+                name: Lint, Type Check and Build React Native SDK
+                runs-on: ubuntu-latest
+
+                steps:
+                - name: Checkout code
+                        uses: actions/checkout@v4
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)